### PR TITLE
Editorial Base Template

### PR DESCRIPTION
**Pull request by tpreusse**
A Mdast Schema for Republik

Most important part: `src/templates/Editorial/index.js`

See documentation live: https://r-styleguide-pr-39.herokuapp.com/templates/editorial
screen shot 2017-11-23 at 19 17 27

### CODE REVIEW

File: src/templates/Editorial/index.js [hunk 1]
```diff
@@ -0,0 +1,392 @@
...
+  InfoBoxFigure
+} from '../../components/InfoBox'
+
+import {
+  matchType,
+  matchZone,
+  matchHeading,
+  matchParagraph,
+  matchImageParagraph
+} from 'mdast-react-render/lib/utils'
```
Comment (lukasbuenger) [inline src/templates/Editorial/index.js]: Is there a reason why `mdast-react-render` is not listed as dependency?

File: src/components/PullQuote/Text.js [hunk 2]
```diff
@@ -12,13 +12,21 @@ const styles = {
...
+    }
   })
 }
 
 export const Text = ({ children, attributes, isQuoted = true }) => {
   return (
-    <div {...attributes} {...styles.text}>
-      {isQuoted ? <span>«{children}»</span> : <span>{children}</span>}
+    <div {...attributes} {...styles.text} {...((isQuoted && styles.quoted) || {})}>
+      <span>{children}</span>
```
Comment (uxengine) [inline src/components/PullQuote/Text.js]: The `<span>` is obsolete now.

File: src/templates/Editorial/index.js [hunk 3]
```diff
@@ -0,0 +1,392 @@
...
+                  component: PullQuoteSource,
+                  editorModule: 'paragraph',
+                  editorOptions: {
+                    type: 'QUOTECITE',
+                    placeholder: 'Quellenangabe / Autor'
+                  },
+                  rules: [
+                    br,
+                    link
+                  ]
```
Comment (uxengine) [inline src/templates/Editorial/index.js]: No action required now, but just for the record:
As a left-wing, this code is veering way too much to the right IMHO ;) 
We might wanna consider breaking things down into smaller objects, and composing the main schema from these.

Comment (lukasbuenger) [reply src/templates/Editorial/index.js]: Word

**Pull request merged**